src/main/scala/Main.scala

```diff
@@ -1,7 +1,26 @@
-object Main extends App {
+import generate.database.PostgresGenerator
+import generate.database.ast._
+import utils.FileUtils
 
-  def square(x: Int): Int =
-    x * x
+object Main extends App {
 
   println("Hello, Temple!")
+
+  val statement = Create(
+    "Users",
+    List(
+      StringColumn("username"),
+      StringColumn("email"),
+      StringColumn("firstName"),
+      StringColumn("lastName"),
+      DateTimeTzColumn("createdAt"),
+      IntColumn("numberOfDogs"),
+      BoolColumn("yeets"),
+      FloatColumn("currentBankBalance"),
+      DateColumn("birthDate"),
+      TimeColumn("breakfastTime")
+    )
+  )
```

Comment:
Should this be here rather than in a test?

Reply:
It means the program can be ran and get meaningful output - I can delete if not worth